src/chapters/saml-auth.xml

```diff
@@ -0,0 +1,176 @@
+                <filename>guacamole-auth-saml-1.2.0.jar</filename>, which must
+            ultimately be placed in <filename>GUACAMOLE_HOME/extensions</filename>.</para>
+    </section>
+    <section xml:id="installing-saml-auth">
+        <title>Installing SAML authentication</title>
+        <para>Guacamole extensions are self-contained <filename>.jar</filename> files which are
+            located within the <filename>GUACAMOLE_HOME/extensions</filename> directory.
+                <emphasis>If you are unsure where <varname>GUACAMOLE_HOME</varname> is located on
+                your system, please consult <xref linkend="configuring-guacamole"/> before
+                proceeding.</emphasis></para>
+        <para>To install the SAML authentication extension, you must:</para>
+        <procedure>
+            <step>
+                <para>Create the <filename>GUACAMOLE_HOME/extensions</filename> directory, if it
+                    does not already exist.</para>
+            </step>
+            <step>
+                <para>Copy <filename>guacamole-auth-saml-1.2.0.jar</filename> within
+                        <filename>GUACAMOLE_HOME/extensions</filename>.</para>
+            </step>
+            <step>
+                <para>Configure Guacamole to use SAML authentication, as described
+                    below.</para>
+            </step>
+        </procedure>
+        <section xml:id="guac-saml-config">
+            <title>Configuring Guacamole for SAML Authentication</title>
+            <indexterm>
+                <primary>configuring SAML authentication</primary>
+            </indexterm>
+            <indexterm>
+                <primary>SAML authentication</primary>
+                <secondary>configuration</secondary>
+            </indexterm>
+            <para>The SAML authentication extension provides several configuration properties
+                to set it up to talk to the IdP.  The SAML IdP also must be configured with
+                Guacamole as a Service Provider (SP).  Configuration of the SAML IdP is beyond
+                the scope of this document, and will vary widely based on the IdP in use.</para>
+            <variablelist>
+                <varlistentry>
+                    <term><property>saml-idp-metadata-url</property></term>
+                    <listitem>
+                        <para>The URI of the XML metadata file that from the
+                            SAML Identity Provider that contains all of the
+                            information the SAML extension needs in order to
+                            know how to authentication with the IdP. This URI
+                            can either be a remote server (e.g. https://) or a
+                            local file on the filesystem (e.g. file://). If this
+                            file is provided via this configuration property, no
+                            other configuration is required.  The metadata file is
```

Comment:
> ... If this file is provided via this configuration property, no other configuration is required. ...

What about `saml-entity-id` and `saml-callback-url`? Aren't those required in all cases?

Reply:
Sounds like they shouldn't be, so I think some additional rewording, here, and a couple of changes to the SAML `ConfigurationService` code are in order.